### PR TITLE
Added UI for listing contacts

### LINKED ISSUE #7: Create list contacts UI

### PR DESCRIPTION
**Pull request by vysakh-poulose**
Fixes #7

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Table/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  return (
+    <div className="flex items-center space-x-3">
+      <Avatar
+        user={{
+          name,
+          imageUrl: name === "Jacob Jones" && "https://i.pravatar.cc/300",
+        }}
+      />
+      <div>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table/Profile.jsx:17]: Use a span here instead of a div.

**Review by yedhink — COMMENTED** [review]
@vysakh-poulose

File: app/javascript/src/components/Dashboard/Contacts/Table/Profile.jsx (hunk 1 above)
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Contacts/Table/Profile.jsx:17]: Fixed in [a6a0152](https://github.com/vysakh-poulose/neeto-ui-challenge-by-vysakh-poulose/pull/16/commits/a6a015270bc9f89e67d1b63dd49b9739dd03cab9)

File: app/javascript/src/components/Dashboard/Notes/Card.jsx [hunk 2]
```diff
@@ -20,12 +20,12 @@ const Card = ({
...
         <Dropdown buttonStyle="text" icon={MenuVertical} position="bottom-end">
-          <li className="cursor-pointer">{t("notes.card.edit")}</li>
+          <li className="cursor-pointer">{t("common.edit")}</li>
           <li
             className="cursor-pointer"
             onClick={() => setIsDeleteOpen(prev => !prev)}
           >
-            {t("notes.card.delete")}
+            {t("common.delete")}
           </li>
```
Comment (navaneethsdk) [inline app/javascript/src/components/Dashboard/Notes/Card.jsx:29]: Can't we use `Dropdown.MenuItem` here? Is there a reason why you are using `li`?

**Review by navaneethsdk — COMMENTED** [review]
@vysakh-poulose

File: app/javascript/src/components/Dashboard/Notes/Card.jsx (hunk 2 above)
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Notes/Card.jsx:29]: Fixed in [b9672d8](https://github.com/vysakh-poulose/neeto-ui-challenge-by-vysakh-poulose/pull/16/commits/b9672d8e6445b1e4b001ad31a64d2f8efca78ff8)

**Pull request merged**